### PR TITLE
go-parquet-tools 1.25.9 (new formula)

### PR DESCRIPTION
**Pull request by hangxie**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/g/go-parquet-tools.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  sha256 "61df81b68b58cfccd6b819bc3edf50e6ecf58e294195167517d71076fb047fd3"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  resource("test-parquet") do
+    url "https://github.com/hangxie/parquet-tools/raw/v1.25.9/testdata/good.parquet"
+    sha256 "daf5090fbc5523cf06df8896cf298dd5e53c058457e34766407cb6bff7522ba5"
+  end
+
```
Comment (chenrui333) [inline Formula/g/go-parquet-tools.rb:14]: this can be moved down into the test block

File: Formula/g/go-parquet-tools.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    system "go", "build", "-ldflags", "-s -w -X github.com/hangxie/parquet-tools/cmd.version=v#{version} -X github.com/hangxie/parquet-tools/cmd.build=#{Time.now.iso8601} -X github.com/hangxie/parquet-tools/cmd.gitHash=3d1f9fb -X github.com/hangxie/parquet-tools/cmd.source=source", *std_go_args, "-o", "#{bin}/parquet-tools"
+  end
```
Comment (chenrui333) [inline Formula/g/go-parquet-tools.rb:18]:
```suggestion
  def install
    ldflags = %W[
      -s -w
      -X github.com/hangxie/parquet-tools/cmd.version=v#{version}
      -X github.com/hangxie/parquet-tools/cmd.build=#{time.iso8601}
      -X github.com/hangxie/parquet-tools/cmd.gitHash=#{tap.user}
      -X github.com/hangxie/parquet-tools/cmd.source=source
    ]
    system "go", "build", *std_go_args(ldflags:, output: bin/"parquet-tools")
  end
```

File: Formula/g/go-parquet-tools.rb (hunk 1 above)
Comment (hangxie) [reply Formula/g/go-parquet-tools.rb]: addressed.

File: Formula/g/go-parquet-tools.rb (hunk 2 above)
Comment (hangxie) [reply Formula/g/go-parquet-tools.rb]: addressed.

**Review by nandahkrishna — APPROVED** [review]
Thank you for your contribution!

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@hangxie, thanks for your first contribution to Homebrew-core! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**